src/parkapi_sources/converters/herrenberg_bike/converter.py

```diff
@@ -0,0 +1,77 @@
+"""
+Copyright 2024 binary butterfly GmbH
+Use of this source code is governed by an MIT-style license that can be found in the LICENSE.txt.
+"""
+
+from abc import ABC
+
+import requests
+from validataclass.exceptions import ValidationError
+from validataclass.validators import DataclassValidator
+
+from parkapi_sources.converters.base_converter.pull import GeojsonInput, PullConverter
+from parkapi_sources.exceptions import ImportParkingSiteException, ImportSourceException
+from parkapi_sources.models import RealtimeParkingSiteInput, SourceInfo, StaticParkingSiteInput
+
+from .models import HerrenbergBikeFeatureInput
+
+
+class HerrenbergBikePullConverter(PullConverter, ABC):
+    geojson_validator = DataclassValidator(GeojsonInput)
+    herrenberg_feature_validator = DataclassValidator(HerrenbergBikeFeatureInput)
+
+    source_info = SourceInfo(
+        uid='herrenberg_bike',
+        name='Stadt Herrenberg - Munigrid: Fahrrad-Abstellanlagen',
+        public_url='https://www.munigrid.de/hbg/dataset/8',
+        source_url='https://r2.munigrid.de/11-d76db68da7d8354bb425c9eb90d6d78a.json',
```

Comment:
This url returns an 404. Looks like it's not stable?

Reply:
I have requested the stable endpoint from the city.